docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md

```diff
@@ -54,6 +54,44 @@ Added a new function that returns initialization data passed when the add-in is
 
 ---
 
+### Mail signature
+
+#### [Office.context.mailbox.item.body.setSignatureAsync](/javascript/api/outlook/office.body?view=outlook-js-preview#setsignatureasync-data--options--callback-)
+
+Added a new function to the `Body` object that adds or replaces the signature in the item body in Compose mode.
+
+**Available in**: Outlook on Windows (connected to Office 365 subscription)
```

Comment:
The Signature APIs might also be available in OWA. akagarwa is working on these APIs for OWA.

Reply:
Can add in a subsequent PR once I receive confirmation.